### PR TITLE
Userguide text

### PR DESCRIPTION
**Pull request by Kobzol**

### CODE REVIEW

File: docs/userguide/text.md [hunk 1]
```diff
@@ -1,144 +1,151 @@
...
+        `"tabular-nums"`,
+        `"diagonal-fractions"` or
+        `"stacked-fractions"`.
+See [SVG documentation](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/font-variant)
+for its meaning.
+
+Each parameter may also be `None` (the default value). This means that the value will be inherited
+from its parent style (see below).
+
+If you want to font or text properties that are not available in `TextStyle`, please
```
Comment (spirali) [inline docs/userguide/text.md:35]: extra "to"

File: docs/userguide/text.md [hunk 2]
```diff
@@ -1,144 +1,151 @@
...
+### Predefined text styles
+There are several predefined global styles available by default. You can use them by their name:
 
-* *"default"* - ``TextStyle(font="Ubuntu", color="black", size=28, line_spacing=1.20, align="middle", variant_numeric="lining-nums")``
-* *"tt"* - ``TextStyle(font="Ubuntu mono")``
-* *"emph"* - ``TextStyle(italic=True)``
-* *"alert"* - ``TextStyle(bold=True, color="red")``
-* *"code"* - ``TextStyle(font="Ubuntu Mono", align="left", color="#222" line_spacing=1.20, size=20)``
-* *"code_lineno"* - ``TextStyle(color="gray")``
+- *"default"* - `TextStyle(font="Ubuntu", color="black", size=28, line_spacing=1.20, align="middle", variant_numeric="lining-nums")`
```
Comment (spirali) [inline docs/userguide/text.md:72]: Ubuntu font is no longer default font (I will update the docs later)

File: docs/userguide/text.md (hunk 1 above)
Comment (Kobzol) [reply docs/userguide/text.md:35]: Actually it was missing `use` :D But good catch, thanks.

File: docs/userguide/text.md (hunk 2 above)
Comment (Kobzol) [reply docs/userguide/text.md:72]: I updated it according to the code.

**Pull request merged**